docs/docs/formatting.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "Formatting"
+route: "/formatting"
+menu: "References"
+---
+
+# File Formatting
+
+Tanka supports formatting for all `jsonnet` and `libsonnet` files using the `tk fmt` command.
+
+By default, the command excludes all `vendor` directories and any files in the immediate directory.
```

Comment:
> and any files in the immediate directory

Could you explain what you mean by this sentence? To me it reads like `tk fmt .` would not format the files in the current directory, which is not true afaict

Reply:
You're quite right! Let me reword this.